api/lib/domain/usecases/create-and-reconcile-user-to-schooling-registration-by-token.js

```diff
@@ -0,0 +1,45 @@
+const { CampaignCodeError, ObjectValidationError } = require('../errors');
+const User = require('../models/User');
+
+module.exports = async function createAndReconcileUserToSchoolingRegistrationByToken({
+  campaignCode,
+  token,
+  birthdate,
+  campaignRepository,
+  userReconciliationService,
+  tokenService,
+  userRepository,
+  schoolingRegistrationRepository,
+  obfuscationService,
+}) {
+  const campaign = await campaignRepository.getByCode(campaignCode);
+  if (!campaign || !campaign.organizationId) {
+    throw new CampaignCodeError();
+  }
+
+  const userDataFromToken = await tokenService.extractUserFromStudentReconciliationToken(token);
```

Comment:
```suggestion
  const externalUser = await tokenService.extractUserFromStudentReconciliationToken(token);
```
Et voir renomme dans tokenService

Reply:
Done